passhportd/app/views_mod/user/user.py

```diff
@@ -49,7 +49,7 @@ def try_ldap_login(login, password):
     s = Server(config.LDAPURI, port=config.LDAPPORT,
                use_ssl=False, get_info=ALL)
     # 1. connection with service account to find the user uid
-    uid = useruid(s, escape_rdn(login))
+    uid = useruid(s, escape_filter_chars(login))
```

Comment:
I've just replaced the function names but I think this one is redundant.

Reply:
It is but better to keep it in case someone copy-paste for another project ;) thanks!

Reply:
The thing is that, following that flow, `login` would go through `escape_filter_chars` twice.

```python3
>>> from ldap3.utils.conv import escape_filter_chars
>>> escape_filter_chars("test ()|")
'test \\28\\29|'
>>> escape_filter_chars(escape_filter_chars("test ()|"))
'test \\5c28\\5c29|'
```

Reply:
I'd get rid of the one pointed here to maintain what the user has provided in case it's going to be used somewhere else in the future.

Reply:
@elg